src/Dev/Install/Installer.php

```diff
@@ -229,9 +228,15 @@ public function index() {
 
             // Create default administrator user and group in database
             // (not using Security::setDefaultAdmin())
-            $adminMember = DefaultAdminService::singleton()->findOrCreateDefaultAdmin();
-            $adminMember->Email = $config['admin']['username'];
-            $adminMember->Password = $config['admin']['password'];
+            $username = $config['admin']['username'];
+            $password = $config['admin']['password'];
+            $adminMember = DefaultAdminService::singleton()
+                ->findOrCreateAdmin(
+                    $username,
+                    _t(DefaultAdminService::class . '.DefaultAdminFirstname', 'Default Admin')
```

Comment:
It just won't collect. It'll still collect from the string in defaultadminservice though.